style.css

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Please replace list bullets with images as are on the mockup
![image](https://user-images.githubusercontent.com/104353194/168749011-5a7c7778-dc79-4f46-bad9-9aadec67837c.png)

![image](https://user-images.githubusercontent.com/104353194/168748939-f601f1b9-2b17-4483-a491-67ed8fb414be.png)